apps/common/components/Header.tsx

```diff
@@ -92,7 +92,7 @@ function NetworkSelector({networks}: {networks: number[]}): ReactElement {
 		<div key={safeChainID} className={'relative z-50 mr-4'}>
 			<Listbox
 				value={safeChainID}
-				onChange={async (value: any): Promise<void> => onSwitchChain(value.value)}>
+				onChange={async (value: number): Promise<void> => onSwitchChain(value)}>
```

Comment:
Unsure about this, it conflicts with https://github.com/yearn/yearn.fi/pull/304/files#diff-110219777e6710fd004b7cacde9e6da2a8eb9d9646ba582321aa88ebc90cdf9eR112